RELEASE_NOTES.md

```diff
@@ -1,6 +1,15 @@
 Release Notes
 =============
 
+v1.0.0-alpha.4
```

Comment:
[fix] Otherwise its not clear that this hasn't been released, so the next person would think to create a new header for the next change.

```suggestion
v1.0.0-alpha.x
```

Reply:
04e9abe